src/main/java/frc/robot/subsystems/AssistedBalance.java

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) FIRST and other WPILib contributors// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+ 
+package frc.robot.subsystems;
+ 
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import edu.wpi.first.wpilibj.Joystick;
+
+//import com.kauailabs.navx.frc.AHRS;
+//import edu.wpi.first.wpilibj.SPI;
+ 
+public class AssistedBalance extends SubsystemBase {
+ 
+   boolean LOCKED = false;
+   private final SwerveSubsystem swerveSubsystem;
+   private final SimulationSubsystem simulationSubsystem;
+   //private AHRS gyro = new AHRS(SPI.Port.kMXP);
+   private Joystick joystick;
+  
+   public AssistedBalance(SwerveSubsystem swerveSubsystem, SimulationSubsystem simulationSubsystem) {
+       this.swerveSubsystem = swerveSubsystem;
+       this.simulationSubsystem = simulationSubsystem;  
+       joystick = new Joystick(0);
+   }
+ 
+   @Override
+   public void periodic() {
+
+    SmartDashboard.putString("|Rotation|", swerveSubsystem.getRotation2d().toString());
+    SmartDashboard.putString("Balancing", "No");
+    if (joystick.getRawButton(1)) { //Button on keyboard is Z
```

Comment:
Need to set this up in OI like other buttons. Then setup the trigger in RobotContainer. Probably want [toggleOnTrue](https://docs.wpilib.org/en/stable/docs/software/commandbased/binding-commands-to-triggers.html#toggleontrue) type of trigger.

Reply:
I have created a toggleOnTrue function in the RobotContainer Class for this command. There is no longer a function like this in the code.